src/services/posAPI.js

```
@@ -1,64 +1,496 @@
+    return await fn(...args);
+  } catch (error) {
+    if (error instanceof ConcurrencyError || error instanceof TransactionError) {
+      throw error;
+    }
+    handleError(error, {
+      showToast: true,
+      context: operationName
+    });
+    throw error;
+  }
+};
```

Biome (1.9.4)

[error] Shouldn't redeclare 'withErrorHandling'. Consider to delete it or rename it.

'withErrorHandling' is defined here:


(lint/suspicious/noRedeclare)